scripts/checks.sh

```diff
@@ -69,10 +69,12 @@ function test_wifi()
 	wifi_active_interfaces=$(nmcli --fields TYPE,STATE device status | grep -c '^wifi *connected')
 	if (( wifi_active_interfaces > 0 )); then
 		local -i wifi_strength
-		wifi_strength=$(nmcli -f IN-USE,SIGNAL device wifi list | awk '/^\*/{print $2}')
-		if (( wifi_strength < wifi_threshold )); then
-			echo "${FUNCNAME[0]}: Configured wifi network has a weak signal (<${wifi_threshold}%)"
-		fi
+		for wifi_strength in $(nmcli -f IN-USE,SIGNAL device wifi list | awk '/^\*/{print $2}'); do
+			if (( wifi_strength < wifi_threshold )); then
+				echo "${FUNCNAME[0]}: Configured wifi network has a weak signal (<${wifi_threshold}%)"
+				return
```

Comment:
A bit confused about `return`

I guess we have 1 field we can return. So if we find any wifi interface with low signal we'll give the same message back?

Reply:
I've updated the code to take into account each individual active wifi connection separately. There's still a tricky bit in that devices acting as APs report a signal strength of 0, though it seems safe to filter those out given that an active connection _must_ have a signal level >0.